scripts/deploy-dao.ts

```diff
@@ -0,0 +1,138 @@
+import { AzoriusTxBuilder } from "../DaoBuilder/AzoriusTxBuilder";
+import { ethers } from "hardhat";
+
+import { getMasterCopies, getSafeData } from "../DaoBuilder/daoUtils";
+import { deployDecentToken } from "../DaoBuilder/dcntTokenDeploy";
+import { encodeMultiSend } from "../DaoBuilder/utils";
+import { decentDAOConfig } from "../DaoBuilder/dcntDAOConfig";
+import { utils } from "ethers";
+
+async function createDAO() {
+  const {
+    zodiacModuleProxyFactoryContract,
+    fractalAzoriusMasterCopyContract,
+    fractalRegistryContract,
+    keyValuePairContract,
+    linearVotingMasterCopyContract,
+    multisendContract,
+  } = await getMasterCopies();
+
+  const { predictedSafeContract, createSafeTx } = await getSafeData(
+    multisendContract
+  );
+
+  console.log("Master copies fetched");
+  console.table({
+    zodiacModuleProxyFactoryContract: zodiacModuleProxyFactoryContract.address,
+    fractalAzoriusMasterCopyContract: fractalAzoriusMasterCopyContract.address,
+    fractalRegistryContract: fractalRegistryContract.address,
+    keyValuePairContract: keyValuePairContract.address,
+    linearVotingMasterCopyContract: linearVotingMasterCopyContract.address,
+    multisendContract: multisendContract.address,
+  });
+
+  // @todo update to to use frameProvider
+  const [deployer] = await ethers.getSigners();
```

Comment:
I saw in the other deployment use of `ethProvider`. I'm wasn't 100% sure of the utility but have my assumptions. left a note here to ask about whether that will be needed here.